_includes/footer.html

```diff
@@ -5,9 +5,9 @@
         <ul class="text-left">
           <li>&copy; {{ site.time | date: '%Y' }} <span>GitHub</span>, Inc.</li>
           <li><a href="https://github.com/site/terms">Terms</a></li>
-          <li><a href="https://training.github.com/kit/license.html">License</a></li>
+          <li><a href="{{ site.baseurl }}/license.html">License</a></li>
```

Comment:
@hectorsector there doesn't appear to be a `license.html` currently in the `services-web` repo. There is a license.md and a license, but neither of them renders at `services.github.com/license.html`. This is definitely something we should try to fix when we update this submodule!